City-of-Pori/pori-kada
# PORI-317 issue with inline image links

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Go to edit any content page and add an image to the body content area.
3. Right-click on the added image and choose to edit atom preferences
4. Add a link to the image.
5. Save the content, make sure the link on the image works, make sure the image as alt and title set correctly on the image tag (if there isn't correct translation for the title and alt texts it might give strange results so make sure to add those if alt and title texts seem strage)
